src/util/metadata/side_metadata/global.rs

```diff
@@ -1409,9 +1425,19 @@ impl SideMetadataContext {
     /// * `start` - The starting address of the source data.
     /// * `size` - The size of the source data (in bytes).
     /// * `no_reserve` - whether to invoke mmap with a noreserve flag (we use this flag to quarantine address range)
```

Comment:
Need to update the doc comment to mention the new argument

Reply:
Done.